src/pheval/utils.py

```diff
@@ -1,7 +1,92 @@
 """
 Contains all pheval utility methods
 """
+import itertools
+from os import path
+from typing import List
 
+import pandas as pd
+from click import Path
 
-def example():
-    return True
+
+def ensure_file_exists(*files: str):
+    """ensure_file_exists"""
+    for file in files:
+        if not path.isfile(file):
+            raise FileNotFoundError(f"File {file} not found")
+
+
+def ensure_columns_exists(**kwargs):
+    """ensure_columns_exists"""
+    flat_cols = list(itertools.chain(*kwargs.get("cols")))
+    dataframes = kwargs.get("dataframes")
+    if not dataframes or not flat_cols:
+        return
+    err_msg = f"""columns: {", ".join(flat_cols[:-1])} and {flat_cols[-1]} \
+- must be present in both left and right files"""
+    for dataframe in kwargs.get("dataframes", [pd.DataFrame()]):
+        if not all(x in dataframe.columns for x in flat_cols):
+            raise ValueError(err_msg)
+
+
+def intersect_scores(
+    left: Path,
+    right: Path,
+    join_columns: List[str],
+    comparison_columns: List[str],
+    sort_columns: List[str],
+):
+    """intersect_scores"""
+    left_df = pd.read_csv(left, sep="\t")
+    right_df = pd.read_csv(right, sep="\t")
+    ensure_columns_exists(
+        cols=[join_columns, comparison_columns, sort_columns],
+        dataframes=[left_df, right_df],
+    )
+    merged = left_df.merge(right_df, on=join_columns, suffixes=("_left", "_right"))
+    return merged
+
+
+def calc_semsim(
```

Comment:
rename this method to something more meaningful, like `compare_semantic_similarity_profiles`. Dont fear long method names if they help readability.